orchagent/Makefile.am

```diff
@@ -10,7 +16,46 @@ else
 DBGFLAGS = -g -DNDEBUG
 endif
 
-orchagent_SOURCES = main.cpp port.cpp orchdaemon.cpp orch.cpp notifications.cpp routeorch.cpp neighorch.cpp intfsorch.cpp portsorch.cpp copporch.cpp tunneldecaporch.cpp qosorch.cpp bufferorch.cpp mirrororch.cpp fdborch.cpp aclorch.cpp saihelper.cpp
+orchagent_SOURCES = \
```

Comment:
I added .h files so that package will be rebuilt if header changes.
As for wildcard, GNU automake has a whole [page](https://www.gnu.org/software/automake/manual/html_node/Wildcards.html) on why we shouldn't use them.